test/indexer/test_hca_indexer.py

```diff
@@ -681,6 +682,17 @@ def test_cell_line_sample(self):
                 self.assertEqual(one(contents['cell_suspensions'])['organ'], ['blood (child_cell_line)'])
                 self.assertEqual(one(contents['cell_suspensions'])['organ_part'], [None])
 
+    def test_metadata_generator(self):
```

Comment:
As mentioned in today's stand-up, please add some assertions about the `metadata` entry to `indexer.test_hca_indexer.TestHCAIndexer#test_indexing` as well.